src/main/java/pacApp/pacConfig/WebSecurityConfig.java

```diff
@@ -99,6 +99,10 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 .antMatchers("/auth/**").permitAll()
                 .antMatchers("/registration").permitAll()
                 .antMatchers("/h2-console/*").permitAll()
+                .antMatchers("/swagger-ui**").permitAll()
+                .antMatchers("/webjars/**").permitAll()
+                .antMatchers("/swagger-resources/**").permitAll()
+                .antMatchers("/v2/api-docs/**").permitAll()
```

Comment:
Fine for know, but we should either remove swagger from production (only leave it in test) or use some kind of auth to access it. Could you look into that @philsch91